migrations/capcons/capabilitymigration.go

```diff
@@ -37,6 +37,16 @@ type CapabilityMigrationReporter interface {
 		accountAddress common.Address,
 		addressPath interpreter.AddressPath,
 	)
+	MissingBorrowType(
+		accountAddress common.Address,
+		addressPath interpreter.AddressPath,
+	)
+	IssuedStorageCapability(
```

Comment:
Maybe rename `IssuedStorageCapability` to indicate a capability *controller* was issued, e.g. `IssuedStorageCapabilityController`

Reply:
Refactored in https://github.com/onflow/cadence/pull/3514/commits/97ddc3129f3cb6a6ed90f24887f77c21042ddcd1